charts/influxdb3-clustered/values.yaml

```diff
@@ -0,0 +1,222 @@
+image:
+  # Overrides the InfluxDb Clustered OCI image tag
+  tag: ""
+
+imagePullSecrets:
+- name: "gar-docker-secret"
+
+nameOverride: ""
+namespaceOverride: ""
+fullnameOverride: ""
+kubitSingleNamespace: false
+operatorImageVersion: "v0.0.14"
+operatorImageOverride: ""
+
+# set to true if you have previously installed the kubit operator from another source
+skipOperator: false
```

Comment:
I imagine we going to place your internal write up on helm/kubit adoption to the official docs site?

Having a link to that once it is available will be really useful I think here :+1: